runtime/doc/windows.txt

```diff
@@ -263,28 +263,54 @@ left of the Vim window.
 Closing a window
 ----------------
 
+:q[uit]
+:{count}q[uit]
 CTRL-W q						*CTRL-W_q*
 CTRL-W CTRL-Q						*CTRL-W_CTRL-Q*
-:q[uit]		Quit current window.  When quitting the last window (not
-		counting a help window), exit Vim.
-		When 'hidden' is set, and there is only one window for the
-		current buffer, it becomes hidden.
-		When 'hidden' is not set, and there is only one window for the
-		current buffer, and the buffer was changed, the command fails.
-		(Note: CTRL-Q does not work on all terminals)
+		Without {count}: Quit the current window.  If {count} is
+		given quit the {count} window
 
-:q[uit]!	Quit current window.  If this was the last window for a buffer,
-		any changes to that buffer are lost.  When quitting the last
-		window (not counting help windows), exit Vim.  The contents of
-		the buffer are lost, even when 'hidden' is set.
+		When quitting the last window (not counting a help window),
+		exit Vim.
 
+		When 'hidden' is set, and there is only one window for the
+		current buffer, it becomes hidden. When 'hidden' is not set,
```

Comment:
I see why people wanted to rewrite the docs in markdown, it would be much more consistent, plus we could use semantic newlines for easier diffs.